api/python/provisioner/_api_cli.py

```diff
@@ -174,6 +174,7 @@ def auth_init(username, password, eauth='pam'):
     'set_network',
     'set_swupdate_repo',
     'sw_update',
+    'rollback_update',
```

Comment:
Should we change this to
**sw_rollback**

Reply:
addressed